pkg/cmd/importcmd/import.go

```diff
@@ -382,12 +382,22 @@ func (options *ImportOptions) Run() error {
 	}
 	options.AppName = naming.ToValidName(strings.ToLower(options.AppName))
 
+	if !options.DisableDraft {
+		// Disable draft if jenkins-x.yaml is already present
+		jenkinsxFile := filepath.Join(options.Dir, "jenkins-x.yml")
```

Comment:
I am a bit unsure whether the existence of `jenkins-x.yml` falls short as criteria here. There is a lot more happening in `options.DraftCreate` than the creation of `jenkins-x.yml`. I would think that one needs to integrate a bit deeper and do in general better checks and error handling along the various steps of `DraftCreate`. 

@abayer, @daveconde - wdyt?

Some tests would be nice regardless. ;-)

Reply:
yeah, i realize the `jenkins-x.yml` is project or rather Jenkins X installation specific which is why it would need to be adjusted as well...